### DISCUSSION TITLE
can we translate our Mobi models into R script or ODEs

### DISCUSSION THREAD

**Discussion by hainiwen** [Open-Systems-Pharmacology/Forum#1729] [How-Tos]
Hi all,

I have a general question: can we somehow translate a established Mobi model into R script or a file with all your ODEs, so it can be rebuilt or reused in other similar software or packages? For example it always puzzled me if I use a dissolution function for oral medicines, how would that be described in ODEs ...

**Comment by Yuri05** [comment]
At the moment: no. Is planned for the future (https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/296)
You can export the ODEs to Matlab though.
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/b49f2c49-e81e-4a4f-8eab-05c94377a55a)

**Reply by hainiwen** [reply]
Thanks! And looking forward to that!!
